releasenotes/notes/iam_default_project-fcfc73518845f8aa.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    **[IAM]** Make project data source provider return default (`#1665 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1665>`)
```

Comment:
Maybe make it more clear, which project is returned?

Reply:
maybe